### DISCUSSION TITLE
How to change default colour for numbers from Blue-ish to some other colour

### DISCUSSION THREAD

**Discussion by abhishrek** [Textualize/rich#1124] [Q&A]
I just started using this lib, and it automatically prints all the numbers in Blueish color by default.
Its hard to see on a black terminal.
I want to change this default behaviour from blue to some other color, how can i do that ?

I am not using console.print, just regular print from rich
```
from rich import print
print ("%f ", mynumber)
```

<img width="573" alt="Screenshot 2021-03-21 at 3 40 27 PM" src="https://user-images.githubusercontent.com/11253347/111901061-c4012b00-8a5b-11eb-9cf5-164f563906ec.png">

**Comment by willmcgugan** [comment]
see docs on themes

https://rich.readthedocs.io/en/stable/style.html

**Reply by abhishrek** [reply]
Thank you for the prompt reply.
These docs refer to console.print()
Is there any way to do it for the regular print() 
```
from rich import print
print ("%f ", mynumber)
```
